bayesian_testing/experiments/delta_normal.py

```diff
@@ -0,0 +1,298 @@
+from numbers import Number
+from typing import List, Tuple
+import numpy as np
+from bayesian_testing.experiments.base import BaseDataTest
+from bayesian_testing.metrics import eval_delta_normal_agg
+from bayesian_testing.utilities import get_logger
+
+logger = get_logger("bayesian_testing")
+
+
+class DeltaNormalDataTest(BaseDataTest):
+    """
+    Class for Bayesian A/B test for Delta-Normal data (Normal with non-positive values).
+    Delta-normal data is typical case of net profit data where sessions could have 0 values,
+    positive or negative values.
```

Comment:
And just one more question:. For the use case you have, using just normal data does not make much sense? Meaning you have lot of 0 values together with some other values that are normally distributed but all together they would not be normal, right? I suppose that is a good case, I am just asking if I understand it correctly.

Reply:
Hi, sorry for late reply..

I thought about your question and tried to simulate some examples. 

For instance, suppose you have a normal data for net profit with mean 20, sample size 50K, but you also have non-converted people as 0 (50K as well). This means your conversion is 50%. If I plot this combined 100K people, the graph will be as such

![Ekran Resmi 2023-02-21 11 17 39](https://user-images.githubusercontent.com/37246409/220287195-d1f7312e-77ab-4c34-8b65-823058e782d2.png)

When I try to fit this distribution to normal, the error rate was too high.

Also, I did a Shapiro-Wilk test, and rejected the hypothesis that this distribution is Normal.

![Ekran Resmi 2023-02-21 11 18 48](https://user-images.githubusercontent.com/37246409/220287431-bdd57b92-66bd-4b0d-95c6-a8758ac35b3e.png)

Actually, it is clear for me now that we need to use a combined distribution (deltanormal). But if you have other suspicions, we can talk about it more!

Reply:
yes this make sense. thanks for details!
(I can resolve this comment, but please have a look at the rest)

Reply:
Ofc, I will be updating them as soon as possible!